lib/MySQL_Session.cpp

```diff
@@ -1003,6 +995,11 @@ void MySQL_Session::generate_proxysql_internal_session_json(json &j) {
 			j["backends"][i]["stream"]["DSS"] = _myds->DSS;
 			if (_myds->myconn) {
 				MySQL_Connection * _myconn = _myds->myconn;
+				for (auto idx = 0; idx < SQL_NAME_LAST; idx++) {
+					if(mysql_tracked_variables[idx].special_handling) {
```

Comment:
We decided that `special_handling` probably needs to removed, but we are keeping it there for future enhancements.
This `if` seems incorrect.